### PR TITLE
Add customization for Connection Monitor V1/V2 in SDK

### PR DESCRIPTION
**Pull request by hofattahMicrosoft**
The following SDK code changes add a customization APIs that are to be used by powershell cmdlets that can keep the customers using the connection monitor V1 APIs. This is to keep backward compatibility for users if they want to use either connection monitor V1 or connection monitor V2.

### CODE REVIEW

File: sdk/network/Microsoft.Azure.Management.Network/src/Microsoft.Azure.Management.Network.csproj [hunk 1]
```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.Network</PackageId>
 		<Description>Provides management capabilities for Network services.</Description>
 		<AssemblyName>Microsoft.Azure.Management.Network</AssemblyName>
-		<Version>19.17.1-preview</Version>
+		<Version>20.0.0-preview</Version>
```
Comment (number213) [inline sdk/network/Microsoft.Azure.Management.Network/src/Microsoft.Azure.Management.Network.csproj]: Please revert this change. We'll update SDK version later with the new release

File: sdk/network/Microsoft.Azure.Management.Network/src/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Network Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Network management functions for managing the Microsoft Azure Network service.")]
 
-[assembly: AssemblyVersion("19.5.0.0")]
-[assembly: AssemblyFileVersion("19.17.1.0")]
+[assembly: AssemblyVersion("20.0.0.0")]
```
Comment (number213) [inline sdk/network/Microsoft.Azure.Management.Network/src/Properties/AssemblyInfo.cs]: Please revert this change. We'll update SDK version later with the new release

**Review by anton-evseev — CHANGES_REQUESTED** [review]

**Review by anton-evseev — APPROVED** [review]

**Comment by hofattahMicrosoft** [comment]
Hi Sdk team,

Can you approve this PR for SDK?

Thanks
Hossam

**Comment by hofattahMicrosoft** [comment]
Hello Erich:

Can you please approve this SDK PR?

It has customization APIs for Connection monitors V1/V2?

https://github.com/Azure/azure-sdk-for-net/pull/9255

Thanks
Hossam

**Review by isra-fel — APPROVED** [review]

**Comment by hofattahMicrosoft** [comment]
Hello Yeming,

Can you please approve this PR:

https://github.com/Azure/azure-sdk-for-net/pull/9313

It small change that removes validation not needed any more for CmV2.

Thanks
Hossam

**Pull request merged**